src/map/itemdb.cpp

```diff
@@ -514,6 +514,10 @@ bool itemdb_isstackable2(struct item_data *id)
 	}
 }
 
+int itemdb_inventory_slot_needed(item_data *id, int amount) {
```

Comment:
Since this is a general function:
Please add a stackable call as well in case someone calls the function without checking before.

Reply:
And since we use C++ now. Maybe change it to call by reference. That way nullptr for id is not possible.